sr_port/anticipatory_freeze.h

```diff
@@ -403,27 +374,58 @@ MBSTART {														\
 	DO_LSEEKWRITEASYNC(CSA, DB_FN, FD, 0, BUFF, 0, CR, STATUS, fake_db_enospc, LSEEKWRITE_IS_TO_DB_ASYNC_RESTART);	\
 } MBEND
 
+#ifdef DEBUG
+#define JNL_HANG_TRIGGER		500
+#define JNL_LSEEKWRITE_HANG(CSA)	jnl_lseekwrite_hang(CSA)
+
+static inline boolean_t jnl_lseekwrite_hang(struct sgmnt_addrs_struct *csa)
```

Comment:
Should these static routines be STATICFNDEFs instead with their declarations likewise tagged STATICFNDCL? Else the routine names won't show up in a pro core.

Steve

Reply:
Ah, I see, these are static INLINE so nevermind for now. Just hope they show up correctly.